modules/gitops-argo-rolouts-cli-ovr.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc
+:_mod-docs-content-type: CONCEPT
+[id="gitops-argo-rollouts-cli-overview_{context}"]
+= Argo Rollouts CLI Overview
+
+The Argo Rollouts CLI plugin is not required to use Argo Rollouts resources, because you can instead deploy and monitor a Rollout resource using the {OCP} Console or the `oc` command. However, usage of the Argo Rollouts CLI plugin allows you to handle Argo Rollouts resources and display it directly from the command line interface.
+
```

Comment:
This does not seem to be too user-friendly to include why the user does not need it as the first information. This can be restructured so that we give benefits first, for example:

```suggestion
Argo Rollouts CLI is an optional plugin that allows you to handle and monitor Argo Rollouts resources directly through the command line interface instead of using {OCP} web console or the `oc` command. 
```
Also please keep in mind how to refer to objects: 
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting

Reply:
Fixed.